libnavigation-ui/src/test/java/com/mapbox/navigation/ui/internal/summary/InstructionListPresenterTest.java

```diff
@@ -1,4 +1,4 @@
-package com.mapbox.navigation.ui.summary.list;
+package com.mapbox.navigation.ui.internal.summary;
 
 import android.text.SpannableString;
 
```

Comment:
Why most of the tests here are commented out 🤔?

Reply:
Will be taken care separately in a different ticket.

Reply:
Let's cut a ticket so we don't forget 🙏